.travis.yml

```diff
@@ -1,15 +1,22 @@
 sudo: true
 language: go
-go:
-  - 1.9
-dist: trusty
+sudo: required
```

Comment:
Doesn't that overload the previous `sudo` parameter? If so, does that mean that the `sudo: true` line can be removed?